Overlay/style.css

```diff
@@ -0,0 +1,12 @@
+@import '../variables.css';
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: var(--outer-space-ultra-dark);
```

Comment:
Hm, yeah! I think this would suit being the `background-color` for our `Tooltip` component we'll make. 😄